### ISSUE TITLE
Test Mode

### ISSUE THREAD

**Issue by sonnysavage** [waldson/omnipay-rede-rest-api#1]
When I set to test mode and execute `purchase()`. This is what is returned by `getData()`:

```
array (size=2)
  'returnCode' => string '26' (length=2)
  'returnMessage' => string 'Affiliation: Required parameter missing.' (length=40)
```

Also, I have found the [documentation in Portuguese](https://www.userede.com.br/desenvolvedores/images/uploads/manual_integracao_api_rest.pdf), but I am hoping you know where I can get it in English.

**Comment by waldson** [comment]
@sonnysavage Unfortunately there isn't an english version of it. You need test credentials. I have this one (They send the same for everyone): 

**Test PV (merchantId):** 50079557
**Test Token (merchantKey):** 4913bb24a0284954be72c4258e229b86

In the following link you can find a list of credit card numbers that you could use to validate your integration: 
https://drive.google.com/file/d/1raFseZaU4sBSmmbF9FuRfT1sG8171_Ea/view?usp=sharing

To approve a transaction just send an integer value (e.g.: 1.00 works, 1.01 doesn't approve).

Hope it helps.

**Comment by sonnysavage** [comment]
You've been a fantastic help!

Is there a way to check on their system to see the transaction values?

**Comment by waldson** [comment]
@sonnysavage I don't think so, it's only available in production environment (as regular transactions in your account).

**Comment by sonnysavage** [comment]
I just started getting "Token: Invalid token." errors. It was working for a while.
